### ISSUE TITLE
Invoice records do not conform to schema

### ISSUE THREAD

**Issue by matt-zach-s** [singer-io/tap-xero#17]
I experienced an error persisting records from the Xero tap. 

The error message when pushing records to the Stitch target are as follows:

target - CRITICAL Error persisting data for table "invoices": Record 73 did not conform to schema:
target - CRITICAL #: #: no subschema matched out of the total 2 subschemas
target - CRITICAL #: expected: null, found: JSONObject
target - CRITICAL #: extraneous key [PlannedPaymentDateString] is not permitted
.
.
.
INFO Exit status is: Discovery succeeded. Tap failed with code -15. Target failed with code 1 and error message: (same as above)

Is the schema issue with [PlannedPaymentDateString](https://github.com/singer-io/tap-xero/blob/master/tap_xero/schemas/invoices.json#L161)?

**Comment by rewilson** [comment]
So this is my issue, that Matt raised on my behalf. More details: I extracted the invoices directly from the Xero API endpoint. Record 73 turns out to be the first invoice that actually has a PlannedPaymentDate, so I'm assuming that something is just wrong with schema validation there.

I'd be happy to see what's going on in the tap itself, but I can't work out the required config settings to make it run locally.

**Comment by rewilson** [comment]
Can confirm that in JSON format (but not in XML format) there is a `PlannedPaymentDateString` attribute:

```
"DateString": "2014-06-03T00:00:00",
"Date": "/Date(1401753600000+0000)/",
"DueDateString": "2014-07-08T00:00:00",
"DueDate": "/Date(1404777600000+0000)/",
"PlannedPaymentDateString": "2014-07-21T00:00:00",
"PlannedPaymentDate": "/Date(1405900800000+0000)/",
```

What's the process for adding that to the schema and getting it into the Stitch app itself?

**Comment by KAllan357** [comment]
Released version 0.1.15 to the Singer platform.
